File: PYTHON/PROGRAM/Product.py
class Product:  #PENGISIAN KELAS PRODUCT
    #KONSTRUKTOR
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price
        
    #GETTER DAN SETTER
    def set_idProduct(self, idProduct):
        self.idProduct = idProduct
    
    def get_idProduct(self):
        return self.idProduct
    
    def set_name(self, name):
        self.name = name
    
    def get_name(self):
        return self.name
    
    def set_brand(self, brand):
        self.brand = brand
    
    def get_brand(self):
        return self.brand
    
    def set_price(self, price):
        self.price = price
    
    def get_price(self):
        return self.price
        

File: PHP/PROGRAM/Product.php
<?php
class Product{
    private $idProduct; //deklarasi objek - objek class product
    private $name;
    private $brand;
    private $price;

    //konstruktor ber parameter
    public function __construct($idProduct, $name, $brand, $price)
    {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    //getter dan setter
    public function get_idProduct(){
        return $this->idProduct;
    }

    public function set_idProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    public function get_name(){
        return $this->name;
    }

    public function set_name($name){
        $this->name = $name;
    }

    public function get_brand(){
        return $this->brand;
    }

    public function set_brand($brand){
        $this->brand = $brand;
    }

    public function get_price(){
        return $this->price;
    }

    public function set_price($price){
        $this->price = $price;
    }
}

?>